sensor/kubernetes/listener/resources/convert.go

```diff
@@ -451,17 +452,17 @@ func (w *deploymentWrap) resetPortExposureNoLock() {
 	}
 }
 
-func (w *deploymentWrap) updatePortExposureFromStore(store *serviceStore) {
-	w.mutex.Lock()
-	defer w.mutex.Unlock()
-
-	w.resetPortExposureNoLock()
-
-	svcs := store.getMatchingServicesWithRoutes(w.Namespace, w.PodLabels)
-	for _, svc := range svcs {
-		w.updatePortExposureUncheckedNoLock(svc)
-	}
-}
+//func (w *deploymentWrap) updatePortExposureFromStore(store *serviceStore) {
+//	w.mutex.Lock()
+//	defer w.mutex.Unlock()
+//
+//	w.resetPortExposureNoLock()
+//
+//	svcs := store.getMatchingServicesWithRoutes(w.Namespace, w.PodLabels)
+//	for _, svc := range svcs {
+//		w.updatePortExposureUncheckedNoLock(svc)
+//	}
+//}
 
 func (w *deploymentWrap) updatePortExposureFromServices(svcs ...serviceWithRoutes) {
```

Comment:
Maybe this method isn't needed anymore either. Any parent call with services can do the same thing with the exposure object.